src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -260,6 +272,7 @@ ss::future<response_ptr> create_topics_handler::handle(
               std::vector<cluster::topic_result> c_res) mutable {
           // Append controller results to validation errors
           append_cluster_results(c_res, response.data.topics);
+          append_topic_properties(ctx, response);
```

Comment:
conflict was here. structure of the surrounding continuation has changed somewhat since 23.1.